ccm_web/client/src/pages/Home.tsx

```diff
@@ -18,48 +22,101 @@ const useStyles = makeStyles((theme) => ({
   },
   featureCardContainer: {
     padding: 5
+  },
+  courseName: {
+    textAlign: 'left',
+    padding: 5
   }
 }))
 
-function FeatureGroup (featureGroup: FeatureUIGroup): JSX.Element {
-  const classes = useStyles()
-  return (
-    <Grid key={featureGroup.id} container item xs={12} spacing={0}>
-      <Grid item xs={12}><Typography variant='h6' className={classes.title} >{featureGroup.title}</Typography></Grid>
-      <Grid container item xs={12}>
-        {featureGroup.features.sort((a, b) => (a.data.ordinality < b.data.ordinality) ? -1 : 1).map(feature => {
-          return (
-            <Grid key={feature.data.id} item className={classes.featureCardContainer} xs={12} sm={4}>
-              <FeatureCard {...feature} />
-            </Grid>
-          )
-        })}
-      </Grid>
-    </Grid>
-  )
+interface HomeProps extends LtiProps {
+  ltiKey: string | undefined
 }
 
-function Home (): JSX.Element {
+function Home (props: HomeProps): JSX.Element {
+  const [globals, isAuthenticated, loading, error] = useGlobals(props.ltiKey)
```

Comment:
Though this works, couldn't we also pass the whole `globals` object in as a prop, since it's available in `App`? That saves us an API call.

Reply:
@chrisrrowland, I'd like to at minimum discuss this before I approve. If you convince me it's okay as is, I'll approve then.

Reply:
I thought there was some magic to useGlobals that was caching information. I can change the how the code works to avoid more instances of useGlobals

Reply:
Not sure what magic you mean. `useGlobals` just makes the call once, compartmentalizes some of the globals fetching, processing, and authentication check logic.